### PR TITLE
Fix for inconsistent column geometry's top cap winding order

### PR DESCRIPTION
**Pull request by igorDykhta**
For #5491

#### Background
Looks like degenerate triangles for triangle strips are used badly and swap winding order when the column isn't extruded.
The correct way to preserve winding order is to add the last vertex from the previous strip and the first vertex from the next strip. 

#### Change List
- Added one additional vertex to correctly create degenerate triangles and adjusted cap triangles generation.

- [x] update tests & render tests

### CODE REVIEW

File: modules/layers/src/column-layer/column-geometry.js [hunk 1]
```diff
@@ -66,8 +66,18 @@ function tesselateColumn(props) {
         i += 3;
       }
     }
+
+    // duplicate the last vertex to create proper degenerate triangle.
+    positions[i + 0] = positions[i - 3];
```
Comment (Pessimistress) [inline modules/layers/src/column-layer/column-geometry.js:71]: Why is this needed? The last two vertices were bottom v0 and top v0, and the next vertex will be top v0.

Comment (igorDykhta) [reply modules/layers/src/column-layer/column-geometry.js:71]: > What do we need to put in between, in order to link up the triangles? We’ll need an even number of new triangles in order to preserve the winding. We can do this by repeating the last vertex of the first row, and the first vertex of the second row

https://www.learnopengles.com/tag/degenerate-triangles/

File: test/render/test-cases/column-layer.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
 import {ColumnLayer, GridCellLayer} from '@deck.gl/layers';
+import GL from '@luma.gl/constants';
 import {hexagons, worldGrid} from 'deck.gl-test/data';
 
+const parameters = {
```
Comment (Pessimistress) [inline test/render/test-cases/column-layer.js:5]: Call this something like `cullBackParameters` so that we know what it's for

File: test/render/test-cases/h3-layers.js [hunk 3]
```diff
@@ -1,7 +1,13 @@
 import {H3HexagonLayer, H3ClusterLayer} from '@deck.gl/geo-layers';
+import GL from '@luma.gl/constants';
 
 import * as h3 from 'h3-js';
 
+const parameters = {
```
Comment (Pessimistress) [inline test/render/test-cases/h3-layers.js:6]: I do not think you need to add culling to `H3HexagonLayer` and `HexagonLayer`. They eventually render the `ColumnLayer`. In the spirit of unit tests, the `ColumnLayer` render tests should include:

1. flat with stroke
2. extruded with wireframe

Each combined with

a. default disk geometry
b. externally provided vertices with CCW winding order
c. externally provided vertices with CW winding order

Comment (igorDykhta) [reply test/render/test-cases/h3-layers.js]: I see. The issue with winding appeared only for hexbin layer, not for h3, probably because hexbin used the default disk geometry, and such a case wasn't covered.

Also, I would enable culling for all layers where we expect culling to work. Render tests are a good place to catch such culling issues.

File: examples/layer-browser/src/data-samples.js [hunk 4]
```diff
@@ -267,3 +267,7 @@ function getMeshSampleData([xCount, yCount], spacing) {
 }
 
 export const meshSampleData = getMeshSampleData([10, 10], 120);
+
+export const polygonCW = [[1, 1, 0], [1, -1, 0], [-1, -1, 0], [-1, 1, 0]];
+
+export const polygonCCW = [[1, 0, 0], [0.3, 1, 0], [-0.6, 1, 0], [-1, -0.3, 0], [0, -1, 0]];
```
Comment (Pessimistress) [inline examples/layer-browser/src/data-samples.js:273]: Just use the same polygon reversed and they can share the same golden images

File: examples/layer-browser/src/data-samples.js [hunk 5]
```diff
@@ -267,3 +267,7 @@ function getMeshSampleData([xCount, yCount], spacing) {
 }
 
 export const meshSampleData = getMeshSampleData([10, 10], 120);
+
+export const polygonCW = [[1, 1, 0], [1, -1, 0], [-1, -1, 0], [-1, 1, 0]];
```
Comment (Pessimistress) [inline examples/layer-browser/src/data-samples.js:271]: Move to the render test file since this is only used there.

File: examples/layer-browser/src/data-samples.js (hunk 4 above)
Comment (igorDykhta) [reply examples/layer-browser/src/data-samples.js:273]: Did it to test odd/even number of verts

**Review by Pessimistress — APPROVED** [review]
